server/src/main/java/com/lunchtime/models/Person.java

```diff
@@ -51,13 +50,28 @@
     @Column(name = "created_by")
     private Long createdBy;
 
+
     @Column(name = "modify_at")
     @UpdateTimestamp
     private Instant modifyAt;
 
     @Column(name = "modify_by")
     private Long modifyBy;
 
+    @Enumerated(EnumType.STRING)
+    @Column(name = "status", columnDefinition = "varchar(50) default 'ACTIVE'")
+    private Status status;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinTable(name = "user_roles",
```

Comment:
I think you don't need this table "user_roles".

Reply:
I did not find another option to attach tables, please share the sample or link if there is one to attach tables without adding a side table

Reply:
I meen, that it will be here enough Colomn, you don't need creat extra table.

Reply:
ok we can fix or modify them within the role task